graph_adventure/adv.py

```diff
@@ -39,17 +40,23 @@ def move_player(self):
     current_room = player.currentRoom.id
     direction = self.pick_unexplored()
     # print(f'Current Room {current_room}\ndirection {direction}') # <-- Debugging
+    if direction == None:
```

Comment:
I love your debugging syntax kevin!, it's a great way to check over your code and to see whats missing!. I see a lot of improvement, the last time I've seen your work.